index.js

```diff
@@ -252,9 +252,11 @@ module.exports = function (options) {
       gutil.log('Running Jasmine in Node');
       try {
         var jasmine = new Jasmine(),
-            terminalReporter = require('./lib/terminal-reporter.js').TerminalReporter;
+            SpecReporter = require('jasmine-spec-reporter');
```

Comment:
If we are no longer using `./lib/terminal-reporter.js` could we just delete the file all together?